docs/markdown/Reference-manual.md

```diff
@@ -460,6 +460,10 @@ first one that is found is returned. Meson versions earlier than
 
 Keyword arguments are the following:
 
+- `native` tells whether to find the executable from the system or use
+  the one defined in the cross file, defaults to `false` and is
+  ignored in non-cross builds
+
```

Comment:
I would rephrase this to explicitly state what the action will be when this is set to `true`. It's not clear to me what it's supposed to do as-is.

Does `native: true` mean it will only look for executables that can be run on the build machine? `false` (the default) will cause it to also use the one defined in the cross file? Which one takes priority if the program is on the build machine and also in the cross file?